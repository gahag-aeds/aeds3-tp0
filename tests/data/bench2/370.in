370 185
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 
Add 138 260
Min 168 290
Add 226 348
Sum 237 359
Sub 232 354
Max 240 362
Sub 200 322
Sum 46 168
Sub 131 253
Max 17 139
Add 240 362
Max 31 153
Sub 52 174
Min 75 197
Sub 32 154
Min 7 129
Sub 102 224
Max 101 223
Add 172 294
Min 224 346
Add 44 166
Min 124 246
Sub 15 137
Sum 81 203
Add 178 300
Max 36 158
Add 127 249
Min 134 256
Add 32 154
Min 173 295
Add 157 279
Max 223 345
Sub 93 215
Sum 189 311
Add 212 334
Max 131 253
Sub 181 303
Sum 218 340
Sub 231 353
Max 118 240
Sub 41 163
Min 30 152
Sub 202 324
Max 93 215
Sub 158 280
Sum 233 355
Sub 153 275
Min 13 135
Add 140 262
Sum 131 253
Add 236 358
Max 183 305
Sub 62 184
Max 57 179
Add 70 192
Min 239 361
Add 93 215
Max 3 125
Add 179 301
Max 178 300
Add 229 351
Min 142 264
Add 153 275
Max 129 251
Add 83 205
Min 171 293
Add 12 134
Min 70 192
Sub 195 317
Sum 49 171
Sub 241 363
Sum 236 358
Add 78 200
Sum 195 317
Add 135 257
Max 236 358
Add 234 356
Min 40 162
Sub 55 177
Min 126 248
Add 224 346
Sum 43 165
Add 115 237
Max 38 160
Sub 154 276
Max 238 360
Add 29 151
Sum 246 368
Add 47 169
Max 15 137
Sub 230 352
Min 29 151
Sub 156 278
Max 189 311
Sub 157 279
Sum 46 168
Sub 112 234
Max 222 344
Sub 115 237
Max 59 181
Sub 78 200
Sum 109 231
Sub 101 223
Max 155 277
Sub 110 232
Sum 235 357
Sub 197 319
Sum 97 219
Add 27 149
Sum 4 126
Add 29 151
Min 4 126
Add 101 223
Max 198 320
Add 95 217
Sum 7 129
Sub 242 364
Max 124 246
Add 59 181
Max 42 164
Add 192 314
Min 41 163
Sub 70 192
Sum 154 276
Add 229 351
Min 33 155
Sub 199 321
Min 147 269
Sub 61 183
Max 67 189
Sub 12 134
Max 138 260
Add 229 351
Max 113 235
Add 46 168
Min 91 213
Add 100 222
Max 49 171
Sub 241 363
Max 181 303
Add 107 229
Sum 77 199
Add 238 360
Sum 207 329
Sub 81 203
Max 85 207
Sub 20 142
Max 75 197
Sub 15 137
Max 78 200
Add 116 238
Max 13 135
Add 118 240
Max 239 361
Sub 167 289
Min 153 275
Sub 24 146
Sum 118 240
Sub 52 174
Max 85 207
Sub 238 360
Sum 63 185
Sub 222 344
Sum 219 341
Sub 229 351
Min 74 196
Sub 117 239
Min 136 258
Sub 177 299
Sum 164 286
Sub 69 191
Sum 36 158
Add 47 169
Max 113 235
Sub 49 171
Min 127 249
Sub 4 126
Max 68 190
Add 202 324
Sum 73 195
Sub 199 321
Sum 124 246
Add 85 207
Max 69 191
Add 116 238
