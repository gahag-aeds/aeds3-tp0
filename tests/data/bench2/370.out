0
6383
99
5980
99
99
-1
-2
97
0
0
6008
97
1
2
102
6731
102
6423
98
-2
98
6139
-2
6374
100
96
0
98
101
0
102
3
0
5866
6319
6509
102
-1
3
5807
98
103
6443
100
0
101
6161
100
100
6102
100
6274
5869
5981
0
99
6154
97
102
0
6419
3
-2
100
99
96
-1
100
99
6487
6240
98
99
98
98
101
-2
6452
99
5879
6052
-3
0
6172
5528
98
-1
97
5816
6109
98
